be/src/runtime/plan_fragment_executor.cpp

```diff
@@ -487,21 +487,22 @@ Status PlanFragmentExecutor::get_next_internal(RowBatch** batch) {
     return Status::OK();
 }
 
```

Comment:
Only rename status to new_status?

Reply:
See changes in line 497 `if (_status.ok())`